The root dispersion calculation incorrectly stopped using the peer dispersion
term when testing the acceptability of a NTP sever.

This has been reported as NTP community bug 3093
http://bugs.ntp.org/show_bug.cgi?id=3093
This patch may be removed when upgrading to a version of NTP with that bug
fixed.

--- ntpd/ntp_proto.c
+++ ntpd/ntp_proto.c
@@ -4338,7 +4338,7 @@ peer_unfit(
 	 * plus the increment due to one host poll interval.
 	 */
 	if (   !(peer->flags & FLAG_REFCLOCK)
-	    && root_distance(peer) >= sys_maxdist
+	    && (root_distance(peer) + peer->disp) >= sys_maxdist
 				      + clock_phi * ULOGTOD(peer->hpoll))
 		rval |= TEST11;		/* distance exceeded */
 
